rfp-proposals/rfp-proposal-filexplorer-AHC.md

```diff
@@ -0,0 +1,216 @@
+# Contact: `CTO: futjrn@herc.one`
+# RFP Proposal: `FILExplorer`
+
+**FILExplorer.org: ![Proof of Ownership](https://github.com/herchackathon/devgrants/blob/rfp-filexplorer-AHCteam/rfp-proposals/filexplorer.png) 
+
+**Link to RFP:** [Explorer](https://github.com/filecoin-project/devgrants/blob/master/rfps/rfp-block-explorer.md)
+
+**RFP Category:** `app-dev`
+
+**Proposer:** `Lrgeoemtry`
+
+**Do you agree to open source all work you do on behalf of this RFP and dual-license under MIT and APACHE2 licenses?:** `Yes`
+
+# Project Description
+
+Hi there, we're the AHC Development team. We've been enjoying IPFS for sometime and would like to be a part of this jounrey as it scales. We plan to have our designer whip up some really sleek UI for a Block Explorer we would use (as devs and a use case) as well as one that others interested in the FIL ecosystem would enjoy and understand intuitively.
+- For Milestone 1 we will have the mocks and the basic ui on a staging environment, and hopefully if the FIL docs are good we'll have basic realtime dev data from the FIL network populating the fields. 
+- For Milestone 2 we'd like to work closely with those in charge to ensure Milestone 1 is behind us while improving the useability and complete functionality of the explorer. 
+- For Milestone 3 we'll have everything documented appropiately so anyone can pick up and build it while deciding an appropiate maintainence agreement post build. 
+
+
+## Deliverables
+
+- Fully functional block explorer website, hosted/maintained/managed by your team. Publicly accessible via a `https://filexplorer.org` site.
+  - Amazon EC2 Hosted Instance for Cloud Compute & CloudFlare for DDOS Protection
+  - Good UX and visual design for both Mobile & Desktop 
+  - Real-time updates based on changes in the network (dev / production)
+  - Fast to load across any region
+  - Open-sourced repo for block explorer codebase, dual-licensed under MIT & Apache 2.0
+  - Well-documented codebase so anyone can learn from the application development process (ReadtheDocs)
+  - User-facing documentation for anyone to spin up the fil explorer locally
+
+
+## Detailed Requirements & Constraints
+
+### User Flows `Brought in For Team's Convienence`
+This tool will most often be used by Filecoin developers, token holders, clients, and miners to investigate transactions, debug their nodes, or generally explore the chain. We have done some thinking about the user needs we would like this tool to satisfy. You can use this as a starting point or take this in a totally new direction. Some example user stories:
+
+- *Prospective Client*: I am considering using Filecoin for my data storage needs. I decide to visit the Filecoin block explorer to get a pulse on important storage contract parameters before using the network.
```

Comment:
Do you have examples of the sorts of storage contract parameters you're thinking of?